index.html

```diff
@@ -21,7 +21,7 @@ <h1 class="logo-heading">
   </header>
```

Comment:
- Remove  `` h1 `` tag in your logo.  (The logo (if the design has one) is on the left, and has a link (a tag)).
- Leave your hamburger button is on the right, and has a nav tag after make these changes.
- Add your Logo into your `` nav `` tag.
![Screen Shot 2021-08-17 at 05 21 02](https://user-images.githubusercontent.com/22258274/129709243-d16f1c2f-6fe3-4dec-a758-d0842d57cc07.png)

Reply:
Well solved